src/sync.cpp

```diff
@@ -1226,8 +1219,6 @@ LocalNode* Sync::checkpath(LocalNode* l, string* localpath, string* localname, d
 
                             statecacheadd(l);
 
-                            delete fa;
```

Comment:
I think we should replace this with `fa.reset()` to ensure fa is deleted before calling `notify`

Reply:
replaced - I see notify() can open a path so best to make it work as it did.  thanks!